webapp/channels/src/i18n/en_AU.json

```diff
@@ -4570,7 +4570,6 @@
   "pricing_modal.planSummary.free": "Increased productivity for small teams",
   "pricing_modal.planSummary.professional": "Scalable solutions for growing teams",
   "pricing_modal.plan_label_trialDays": "{days} DAYS LEFT ON TRIAL",
-  "pricing_modal.price.freeForever": "Free forever",
```

Comment:
Do not update loclaization files apart of `en.json` or you break weblate.

Reply:
Oh, I didn't knew that! Reverted now.